src/test/resources/com/jetbrains/snakecharm/cucumber/highlighting/wildcards_inspections.feature

```diff
@@ -0,0 +1,182 @@
+Feature: Check highlighting of inspections on wildcards
```

Comment:
Please don't merge test for 2 different features (inspections) in a single long file